src/client/components/AddFriendModal/AddFriendModal.jsx

```diff
@@ -11,10 +12,10 @@ class AddFriendModal extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      email: ''
+      email: '',
```

Comment:
is this intentional?

Reply:
this is safer when merging. If someone adds another property inside state, it will not break due to lack of comma.